printer_mods/darkestbo/4 channel relay mount/README.md

```diff
@@ -0,0 +1,14 @@
+4 Channel relay mount for 2020 aluminium profile
```

Comment:
Please rename this directory so it doesn't have spaces in the name.  So instead of calling it `4 channel relay mount` rename it to `4_channel_relay_mount`.